lib/decidim/maintainers_toolbox/release_candidate_version.rb

```diff
@@ -0,0 +1,320 @@
+        2. Review and merge this PR
+        3. Once that PR is merged, run the following commands to create the tags and push the gems to RubyGems and the packages to NPM:
+        > git pull
+        > bin/rake release_all
+        4. Usually, at this point, the release branch is deployed to Metadecidim during, at least, one week to validate the stability of the version.
+        EOMESSAGE
+
+        puts "*" * 80
+        puts finish_message
+      end
+
+      # Given a version number, returns the next release candidate
+      #
+      # If the current version number is `dev`, then we return the `rc1` version
+      # If the current version number is `rc`, then we return the next `rc` version
+      # Else, it means is a `minor` or `patch` version. On those cases we raise an Exception, as releases candidates should
+      # be only done from a `dev` or a `rc` version.
+      #
+      # @raise [InvalidVersionTypeError]
+      #
+      # @param current_version_number [String] - The version number of the current version
+      #
+      # @return [String] - the new version number
+      def next_version_number_for_release_candidate(current_version_number)
+        if current_version_number.include? "dev"
+          major, minor, patch = parsed_version_number(current_version_number)
+          new_version_number = "#{major}.#{minor}.#{patch}.rc1"
+        elsif current_version_number.include? "rc"
+          new_rc_number = current_version_number.match(/rc(\d)/)[1].to_i + 1
+          new_version_number = current_version_number.gsub(/rc\d/, "rc#{new_rc_number}")
+        else
+          error_message = <<-EOMESSAGE
+          Trying to do a release candidate version from patch release. Bailing out.
+          You need to do a release candidate from a `dev` or from another `rc` version
+          EOMESSAGE
+          raise InvalidVersionTypeError, error_message
+        end
+
+        new_version_number
+      end
+
+      def next_version_number_for_dev(current_version_number)
+        major, minor, patch = parsed_version_number(current_version_number)
+
+        "#{major}.#{minor.to_i + 1}.#{patch}.dev"
+      end
+
+      def generate_empty_changelog
+        major, minor, patch = parsed_version_number(@old_version_number)
+
+        changelog_contents = <<-EOCHANGELOG
+# Changelog
+
+## [Unreleased](https://github.com/decidim/decidim/tree/HEAD)
+
+Nothing.
+
+...
+
+## Previous versions
+
+Please check [#{major}.#{minor}-stable](https://github.com/decidim/decidim/blob/release/#{major}.#{minor}-stable/CHANGELOG.md) for previous changes.
+EOCHANGELOG
+
+        File.write("CHANGELOG.md", changelog_contents)
+      end
+
+      def generate_empty_release_notes
+        release_notes_contents = <<-EORELEASE
+# Release Notes
+
+## 1. Upgrade notes
+
+As usual, we recommend that you have a full backup, of the database, application code and static files.
+
+To update, follow these steps:
+
+### 1.1. Update your Gemfile
```

Comment:
Since we have the policy of one Decidim version, has one ruby version, i would add here also the `Upgrade ruby step`